PurchasesTests/Identity/IdentityManagerTests.swift

```diff
@@ -0,0 +1,420 @@
+//
```

Comment:
tests start here

Reply:
the ones in this class aren't fully new, the file was moved into the `Identity` group. I'll point out the new ones.